e2e/Utils.ts

```diff
@@ -369,3 +358,5 @@ export const testIfSlAndHg = (...args: Parameters<typeof test>) => {
     test.skip(...args);
   }
 };
+
+export {normalizeIcons} from '@jest/test-utils';
```

Comment:
The reexport is unnecessary. This util is part of the internal testing infrastructure, it isn’t a public API.

```suggestion
```